### ISSUE TITLE
No support for Kubernetes 1.25.* or 1.26.*

### ISSUE THREAD

**Issue by MikkoUronen** [oracle/weblogic-kubernetes-operator#4029]
Based on 

`operator/src/main/java/oracle/kubernetes/operator/helpers/KubernetesVersion.java`

only the following Kubernetes releases are supported. 

...
`private static final String[] MINIMUM_K8S_VERSIONS = {"1.21.10", "1.22.7", "1.23.4", "1.24.0"};`
...

As 1.25.* or 1.26.* is not listed the WLS Kubernetes Operator is not starting eg. on the 1.25.4 which is the latest supported release for Oracle Kubernetes Engine on Oracle Cloud. 

Is there some technical reasons why eg. 1.25.* is not supported? If not could you please add the support as this is now blocking our customers upgrade of the OKE cluster.

**Comment by rjeberhard** [comment]
Operator 4.0.x supports Kubernetes 1.25, as documented here: https://oracle.github.io/weblogic-kubernetes-operator/introduction/prerequisites/introduction/ :
 
```
For the current production release 4.0.5:

Kubernetes 1.21.10+, 1.22.7+, 1.23.4+, 1.24.0+, and 1.25.0+ (check with kubectl version).
```

Since there is no required minimum bug-fix version of 1.25, there was no need to add it to `KubernetesVersion.java`. 

Note: our testing against 1.26 suggests that the operator works against that version as well but we've not yet formally certified, which is why 1.26 is not yet listed in the documentation.

What issue are you seeing when starting the operator? Operator 3.4.x is not supported on 1.25 and should report an error both during the Helm install and starting the container.

**Comment by MikkoUronen** [comment]
Hi,

The only log msg for the operator POD was:

`$ kubectl logs -n wls-operator-ns pod/weblogic-operator-7f6c896886-422sr
Launching Oracle WebLogic Server Kubernetes Operator...
VM settings:
    Max. Heap Size (Estimated): 29.50G
    Using VM: OpenJDK 64-Bit Server VM

{"timestamp":"2023-03-01T09:16:24.387496701Z","thread":1,"fiber":"","namespace":"","domainUID":"","level":"INFO","class":"oracle.kubernetes.operator.helpers.HealthCheckHelper","method":"createAndValidateKubernetesVersion","timeInMillis":1677662184387,"message":"Kubernetes version is: v1.25.4","exception":"","code":"","headers":{},"body":""}`

and then the POD just crashed. 

I was able to workaround the problem by forking the 3.4.6 code base, adding the supported version to the KubernetesVersion.java and doing the required code changes in other classes to get the policy/v1beta1 deprecations handled. There might be other compatibility issues as well but this change allowed us to get the 3.4.* operator up and running again and more time for us to test the 4.0.5 upgrade. I tested the upgrade to 4.0.* months back but then there was some issues I didn't have time to investigate at that time. 

Thanks for the quick response! We'll start planning the 4.0.* upgrade.
